MapboxVisionTests/VisionManagerTests.swift

```diff
@@ -0,0 +1,139 @@
+@testable import MapboxVision
+import XCTest
+
+class VisionManagerTests: XCTestCase {
+    var visionManager: VisionManager?
+    var dependencies: VisionDependencies?
+
+    override func setUp() {
+        super.setUp()
+
+        let native = MockNative()
```

Comment:
Can we pass `MockNative()` into `dependencies` without creating a separate constant (like everything else - `MockSynchronizable()` etc.)?

Reply:
yes, fixed